Update documents with current information. (#79)
cisagov/development-guide: onboarding-checklist.md

#### Mandatory training ####

> - [ ] Complete ~~[PALMS Cybersecurity Training](https://etms.hq.dhs.gov)~~,

Reviewer: PALMS is dead.  I think you can just remove this item entirely.

The new "hotness" is here: [CISA Course Management Site](https://usdhs.sharepoint.com/sites/CISA_OCLO_Intranet/CISACourses)

I think the specific training is here: [Cybersecurity Awareness Training ](https://usdhs.sharepoint.com/sites/CISA_OCLO_Intranet/CISACourses/SitePages/DHS-Mandatory-Training---CSAT-FY22.aspx)

_Note: as soon as you document these URLs they will be rotated to new random values._ 

Feel free to resolve this comment In absentia.

Reviewer 2: Please see commit fe41c6e.

including Mandatory Cyber Security and Privacy Training, and accepting
the Rules of Behavior, which is required before we can give you access to any
systems, like Trio.

> - PALMS has been retired, and is currently being replaced by the

Reviewer 2: ```suggestion
- PALMS has been retired, and is in the process of being replaced by the
```

Author: @jsf9k The current site is an interim replacement until a new final solution is in place.  I can reword it, but don't think your recommendation is accurate

Reviewer 2: I see.  Maybe change "currently" to "temporarily" then?

Reviewer 2: Fixed in commit d434617.

> [CISA Course Management Site](https://usdhs.sharepoint.com/sites/CISA_OCLO_Intranet/CISACourses/SitePages/LearningTeamHome.aspx)

Reviewer 3: Add period at end to match other list items:
```suggestion
[CISA Course Management Site](https://usdhs.sharepoint.com/sites/CISA_OCLO_Intranet/CISACourses/SitePages/LearningTeamHome.aspx).
```

cross-team and cross-agency collaboration and is generally
`first.last@trio.dhs.gov`. Trio may take a few days to get set up.

> - Trio accounts are currently on hold.

Reviewer 2: ```suggestion
*Note that the issuance of Trio accounts is currently on hold.*
```
